src/NServiceBus.Persistence.AzureStorage/Config/AzureSagaPersisterConfig.cs

```diff
@@ -1,52 +0,0 @@
-namespace NServiceBus.Config
-{
-    using System.Configuration;
-
-    /// <summary>
-    /// Config section for the Azure Saga Persister
-    /// </summary>
-    public class AzureSagaPersisterConfig:ConfigurationSection
```

Comment:
I'm bearing in mind that ASP (or P.AS) is a brand new package.
Saying that, it's the same "candy" in a new wrapper. So perhaps for better guidance obsoletes would help.